package.json

```diff
@@ -4,13 +4,15 @@
   "private": true,
   "license": "MIT",
   "scripts": {
-    "build": "nest build",
+    "build": "yarn generate-abis && nest build",
     "format": "prettier --write .",
     "format-check": "prettier --check .",
+    "generate-abis": "node ./scripts/generate-abis.js",
+    "postinstall": "yarn generate-abis",
     "start": "nest start",
     "start:dev": "nest start --watch",
     "start:debug": "nest start --debug --watch",
-    "start:prod": "node dist/main",
+    "start:prod": "node dist/src/main",
```

Comment:
By importing from a folder outside of `src`, Nest creates a "nested" `dist`, e.g.:

```
└── dist
    ├── abis
    └── src
```